src/hotspot/cpu/s390/s390.ad

```diff
@@ -7292,7 +7470,7 @@ instruct negD_reg(regD dst, regD src, flagsReg cr) %{
 
 // Sqrt float precision
 instruct sqrtF_reg(regF dst, regF src) %{
-  match(Set dst (ConvD2F (SqrtD (ConvF2D src))));
+  match(Set dst (SqrtF src));
```

Comment:
Did the old code cause problems? Required nodes not matched? Maybe this should get fixed in a separate PR for backports?

Reply:
[JDK-8308277](https://bugs.openjdk.org/browse/JDK-8308277)  This one was done for RISCV platform. If I do not make this change vectorization for float squreroot (Match.sqrt()) does not happen on s390x.  Unlike RISCV I did not see ConvF2D -> ConvD2F IR Node sequence on s390x.
Shall I raise a separate PR for this. ?

Reply:
I think doing this in a separate PR makes sense. You may want to backport it to 21 and/or 17.

Reply:
I raise separate PR. Thank you

Reply:
I think we should integrate that one first and then merge with this PR. Your updates look good.

Reply:
@TheRealMDoerr  [jdk-8328633](https://github.com/openjdk/jdk/pull/18406) Here is the the PR I have raised for this. 
Could you review an approve it?